lower_limit: 0.0
upper_limit: 1.0

depth_image_topic: /camera_legs/depth_registered/cloud_image
rgb_image_topic: /camera_legs/rgb/image_rect_color
#depth_image_pub: /cv_leg_tracking/raw_image
topic_point_cloud: /camera_legs/depth_registered/points
from_pc2_depth_image_topic: /cv_leg_tracking/from_pc2_depth_image_topic
num_images_for_background: 1
person_distance: 1.5
bottom_factor: 1.
shouldOutput: false
camera_frame_id: camera_depth_frame
usePCL: false
#person_hips: -0.1
#person_neck: 0.4
person_hips: 0.2
hips_height_world: 0.8
camera_height_world: 0.6
person_hips_frame_id: base_link
person_neck: 0.4
useCentroid: true
camera_info_topic: /camera_body/depth_registered/camera_info
camera_angle_radians: -0.26
